File: examples/dotNET/2. Functions/Binary Analysis/Basic Particle/cs/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using NationalInstruments.Vision;
using NationalInstruments.Vision.Analysis;

namespace BasicParticle
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
            imageViewer2.Palette.Type = NationalInstruments.Vision.WindowsForms.PaletteType.Binary;
        }

        public VisionImage BeforeImage
        {
            get { return imageViewer1.Image; }
        }

        public VisionImage AfterImage
        {
            get { return imageViewer2.Image; }
        }
    }
}

File: examples/dotNET/2. Functions/Binary Analysis/Basic Particle/vb/Form2.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports NationalInstruments.Vision
Imports NationalInstruments.Vision.Analysis

Partial Public Class Form2
    Inherits Form
    Public Sub New()
        InitializeComponent()
        imageViewer2.Palette.Type = NationalInstruments.Vision.WindowsForms.PaletteType.Binary
    End Sub

    Public ReadOnly Property BeforeImage() As VisionImage
        Get
            Return imageViewer1.Image
        End Get
    End Property

    Public ReadOnly Property AfterImage() As VisionImage
        Get
            Return imageViewer2.Image
        End Get
    End Property
End Class